### PR TITLE
Removed toast (entities turn on/off and OK service calls)

### PR DESCRIPTION
**Pull request by yosilevy**
Still will show on service error

### CODE REVIEW

**Comment by iantrich** [comment]
yeah, this is needed; but perhaps it should be configurable?

**Comment by yosilevy** [comment]
Discussed with @balloob before implementation and was decided that opt in/out is not required.

**Comment by iantrich** [comment]
fine by me as long as you guys have considered it 👍

**Review by iantrich — APPROVED** [review]
Remove L18 import computeStateName per the linter and this should be good to go, I think.

File: src/layouts/app/connection-mixin.js [hunk 1]
```diff
@@ -65,44 +64,6 @@ export default (superClass) =>
...
-              }
-              if (names.length === 0) {
-                names.push(entityIds[0]);
-              }
-
-              let message;
-              const name = names.join(", ");
-              if (service === "turn_on" && serviceData.entity_id) {
-                message = this.hass.localize(
-                  "ui.notification_toast.entity_turned_on",
```
Comment (balloob) [inline src/layouts/app/connection-mixin.js]: Remove the translation strings too

Comment (yosilevy) [reply src/layouts/app/connection-mixin.js]: Done

**Comment by gavinwoolley** [comment]
I have been debugging for a while why this had vanished till I noticed this PR. 
I found this feature really useful . So I knew I'd actually pressed the button. Some switches take a few seconds to respond and change state etc. 
Will there be any chance you could add a toggle to turn on/off?

**Comment by yosilevy** [comment]
Well it vanished because it is rarely useful and just makes the UI technical and not really simple user friendly. As for adding it back - @balloob ?

**Comment by balloob** [comment]
We should not add it back globally. We can consider adding something back on a per-button case.

**Comment by balloob** [comment]
Generally, switches you will see that they switch. We have also been adding `haptic` events, which we can start using on mobile phones. https://developer.mozilla.org/en-US/docs/Web/API/Navigator/vibrate

**Comment by gavinwoolley** [comment]
I have a lot of scripts which send IR command's to TVs etc . They don't highlight when active. It was just nice to get some positive feedback that I had pressed the button and the command was sent. 
Haptic events sound perfect. Would like to try that.

**Comment by iantrich** [comment]
@gavinwoolley they are currently only available in the iOS companion app beta release

**Pull request merged**